lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,32 @@
+<div id="change-session-<%-entitlementUUID%>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('Change to a different session or leave current session.')%>
+        <% } else { %>
+            <%- gettext('To access the course, select a session.')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select" aria-label="<%- gettext('Session Selection Dropdown')%>">
+            <% _.each(availableSessions, function(session) { %>
+                <option data-session_id="<%-session.session_id%>">
+                    <%-session.session_dates%>
+                    <% if (session.session_id == currentSessionId) { %>
+                        <%- gettext('  - Currently Selected')%>
+                    <% } else if (session.enrollment_end){ %>
+                        <%- gettext(' (Open until ') + session.enrollment_end + ')'%>
+                    <% } %>
```

Comment:
I'm not sure whether these translations will work as expected.

Reply:
Changed